static/app/types/hooks.tsx

```diff
@@ -165,7 +165,7 @@ export type ComponentHooks = {
   'component:disabled-custom-symbol-sources': () => React.ComponentType<DisabledCustomSymbolSources>;
   'component:disabled-member': () => React.ComponentType<DisabledMemberViewProps>;
   'component:disabled-member-tooltip': () => React.ComponentType<DisabledMemberTooltipProps>;
-  'component:enhanced-org-stats': () => React.ComponentType<UsageStatsOrganizationProps>;
+  'component:enhanced-org-stats': () => React.ComponentType<OrganizationStatsProps>;
```

Comment:
```suggestion
  'component:enhanced-org-stats': OrganizationStats;
```
this passes the linter and also works. it's a little weird though because every other component that is a hook is a functional component whereas this one is a class